app/helpers/gov_one_helper.rb

```diff
@@ -31,11 +31,6 @@ def login_button
     govuk_button_link_to t('gov_one_info.sign_in_button'), login_uri.to_s
   end
 
-  # @return [String]
-  def logout_button
-    govuk_button_link_to t('gov_one_info.sign_out_button'), logout_uri.to_s
```

Comment:
Are we also removing the locale/resource? Again, if there are multiple button keys, we can nest them (flips how you read it).